ui/components/common/CommonToolbar.js

```diff
@@ -47,6 +49,8 @@ class CommonToolbar extends Component {
                 type="text"
                 placeholder="Filter by Name"
                 className={toolbarFilter}
+                value={get(this.props.filter, "name", "")}
```

Comment:
Why you need `lodash.get`?

Reply:
Isn't there some other way than decomposing? I really don't like it.

Reply:
I'm sure you'll get used to it after a few tries! Unfortunately, I don't think installing a whole new package just to avoid am unliked built-in feature of ES6 is justified :/

Reply:
ok, then :/